scheduler/core/sources.py

```diff
@@ -101,15 +101,15 @@ def use_file(self,
                 if calendar and gmos_fpu and gmos_gratings:
                     file_resource = FileResourceService()
 
-                    # TODO: files_input is undefined. This will be fixed later.
-                    for site in files_input.sites:
+                    for site in sites:
                         suffix = ('s' if site == Site.GS else 'n').upper()
-                        file_resource.load_files(f'GMOS{suffix}_fpu_barcode.txt',
+                        file_resource.load_files(site,
```

Comment:
So now, the calendar is loaded on a site-by-site basis according to my last PR, so this logic should work; however, we do need to pass in a faults and eng tasks file here. Check `file_resource.load_files` for the new signature.